channels/us_stirr.m3u

```diff
@@ -351,61 +351,61 @@ https://dai.google.com/linear/hls/event/EpqgwRlpQKq73ySVSohJWA/master.m3u8
 https://dai.google.com/linear/hls/event/RoiVzG-4TJiJMSVUatDd4A/master.m3u8
 #EXTINF:-1 tvg-id="STIRRRetroCrush.us" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMjQwMDkzNDRf/RetroCrush_576x576.png" group-title="",STIRR RetroCrush TV (720p)
 https://dai.google.com/linear/hls/event/7LAMGFcmQN6iFJjNoHWXrg/master.m3u8
-#EXTINF:-1 tvg-id="STIRRRevry.us" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",STIRR Revry (720p)
+#EXTINF:-1 tvg-id="STIRRRevry.us" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",STIRR Revry (360p)
 https://dai.google.com/linear/hls/event/gvO6-Y6TTjCxRf1QALU4VQ/master.m3u8
 #EXTINF:-1 tvg-id="STIRRRingofHonor.us" tvg-country="US" tvg-language="English" tvg-logo="https://komonews.com/resources/media2/3x1/full/119/center/90/154876f2-ed78-4a1e-959d-96aab12af9cd-small3x1_STIRR_Logo_0720_ROHBestofthePlanet_1920x1080_EPG.png?cb=c81e728d9d4c2f636f067f89cc14862c" group-title="",STIRR Ring of Honor (720p)
 https://dai.google.com/linear/hls/event/RNiQYO3aTjOqTe8od1zlqA/master.m3u8
 #EXTINF:-1 tvg-id="STIRRSilkStalkings.us" tvg-country="US" tvg-language="English" tvg-logo="https://komonews.com/resources/media2/3x1/full/119/center/90/f391754b-a2f8-4c0e-bdf9-7c40ffe92655-small3x1_stirr_1219_epg_silkstalkings_1920x1080.png?cb=eccbc87e4b5ce2fe28308fd9f2a7baf3" group-title="",STIRR Silk Stalkings (720p) [Offline]
 https://dai.google.com/linear/hls/event/8ZYru1fgSY6JL1Ejb6T5Ag/master.m3u8
 #EXTINF:-1 tvg-id="STIRRSOAR.us" tvg-country="US" tvg-language="English" tvg-logo="https://komonews.com/resources/media2/3x1/full/119/center/90/2b6ed67f-5284-4622-8594-a956633c8383-small3x1_stirr_1219_epg_soartv_1920x1080.png?cb=eccbc87e4b5ce2fe28308fd9f2a7baf3" group-title="",STIRR SOAR (720p)
 https://dai.google.com/linear/hls/event/_PDxBUttQYqkxPnmh3VOZA/master.m3u8
-#EXTINF:-1 tvg-id="STIRRSpace1999.us" tvg-country="US" tvg-language="English" tvg-logo="https://komonews.com/resources/media2/3x1/full/119/center/90/aaaa25c9-d98a-46b8-8dca-3851ab93d3ee-small3x1_STIRR_Logo_0320_SPACE1999_1920x1080_EPG.png?cb=c81e728d9d4c2f636f067f89cc14862c" group-title="",STIRR Space 1999 (768p)
+#EXTINF:-1 tvg-id="STIRRSpace1999.us" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/fr/b/b3/Cosmos_1999.jpg" group-title="",STIRR Space:1999 (768p)
 https://dai.google.com/linear/hls/event/NeKNJHuzSeCiN_7Fcuo83Q/master.m3u8
 #EXTINF:-1 tvg-id="STIRRSports.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/SAfy1us.png" group-title="Sports",STIRR Sports (720p) [Offline]
 https://dai.google.com/linear/hls/event/1B2yihdIR1mCL63rXzERag/master.m3u8
```

Comment:
If you have an alt to this one, since it's offline, it can be interesting.

Reply:
Which is offline? The Space:1999 stream and suggested logo both appear online for me.

Reply:
The Stirr Sports one

Reply:
It seems the "STIRR Sports" channel is gone for good. It's no longer part of the official listing at stirr.com nor a part of Matt Huisman's listing for STIRR.